config/config.go

```diff
@@ -70,13 +70,35 @@ func DefaultConfig() *Config {
 	return &Config{Hazelcast: hz}
 }
 
-func writeToFile(config *Config, confPath string) error {
-	var err error
-	var out []byte
-	if out, err = yaml.Marshal(config); err != nil {
-		return err
-	}
-	return file.CreateMissingDirsAndFileWithRWPerms(confPath, out)
+const defaultUserConfig = `hazelcast:
+  clientname: ""
+  logger:
+    level: error
+  cluster:
```

Comment:
Could you also add the `network` config? Probably `addresses` is adequate.

Reply:
Added it here with connection timeout field https://github.com/hazelcast/hazelcast-commandline-client/pull/103/commits/d6297f724be67ca4d064e4783dae5e668bfba38b